research/Workshop_1_Write_Up.md

```diff
@@ -10,6 +10,7 @@ Our first workshop brought together 16 researchers (undergraduates, doctoral stu
 ### Time Constants 
 #### Members 
 Chen Li (UK, Masters)  
+Peter Crowe (Germany, Undergraduate)
```

Comment:
I noticed that it isn't rendering correctly. There needs to be a space (" ") at the end of the line in Markdown, otherwise it renders like this:

![image](https://user-images.githubusercontent.com/9436784/179394974-7350ea9d-f267-433c-b275-e73ac3705c6e.png)